webapp/src/cardFilter.ts

```diff
@@ -77,43 +81,43 @@ class CardFilter {
             if (filter.values.length === 0) {
                 return true
             }
-            return filter.values[0] === value
+            return filterValue === value
```

Comment:
nit: we can take advantage of TS here and use directly:
```suggestion
            return filter.values?[0].toLowerCase() === value
```

Reply:
Sure, we can directly use `filter.values[0].toLowerCase()` because the above `if` is checking if there is at least one element in the array.